netlify/edge-functions/docs-image.tsx

```diff
@@ -0,0 +1,134 @@
+              fill="#007FFF"
+            />
+          </svg>
+          <div
+            style={{
+              height: 40,
+              width: 2,
+              backgroundColor: '#DAE2ED',
+              margin: '0 24px',
+            }}
+          />
+          <p
+            style={{
+              fontFamily: 'General Sans',
+              fontSize: '20px',
+              fontWeight: 700,
+              lineHeight: '36px',
+              letterSpacing: '0.14em',
+              color: '#007FFF',
+            }}
+          >
+            {product}
+          </p>
+        </div>
+        <div
+          style={{
+            fontFamily: 'General Sans',
+            fontStyle: 'normal',
+            fontWeight: 600,
+            fontSize: '68px',
+            lineHeight: '82px',
+            color: '#0B0D0E',
+            marginLeft: 100,
+            marginRight: 100,
+            marginTop: 300,
+            width: '100%',
+            display: 'flex',
+            flexDirection: 'column',
+            justifyContent: 'flex-start',
+          }}
+        >
+          <div style={{ width: '100%', textAlign: 'center', color: '#007FFF' }}>{folder}</div>
+          <div
+            style={{
+              width: '100%',
+              textAlign: 'center',
+
+              fontFamily: 'General Sans',
+              fontStyle: 'normal',
+              fontWeight: 500,
+            }}
+          >
+            {page}
+          </div>
+        </div>
+      </div>
+    ),
+    {
+      width: 1280,
+      height: 640,
+      fonts: [
+        {
+          name: 'General Sans',
+          data: await fetch('https://fonts.cdnfonts.com/s/85793/GeneralSans-Medium.woff').then(
+            (a) => a.arrayBuffer(),
+          ),
+          weight: 1000,
+          style: 'normal',
+        },
+        {
+          name: 'General Sans',
+          data: await fetch('https://fonts.cdnfonts.com/s/85793/GeneralSans-Semibold.woff').then(
+            (a) => a.arrayBuffer(),
+          ),
+          weight: 600,
+          style: 'normal',
+        },
+        {
+          name: 'General Sans',
+          data: await fetch('https://fonts.cdnfonts.com/s/85793/GeneralSans-Bold.woff').then((a) =>
+            a.arrayBuffer(),
+          ),
+          weight: 700,
+          style: 'normal',
+        },
+      ],
+    },
+  );
+}
+export const config = { path: '/og-docs' };
```

Comment:
Should we prefix the paths to avoid conflicts?

```suggestion
export const config = { path: '/edge-functions/og-docs' };
```

Reply:
We could*. I assumes the use of `og-` would be enough. But no strong opinion on this topic

Reply:
Ah, I meant more in the sense of making it easier to debug. With this folder prefix, less confusion possible, no it's not a Next.js page.